### PR TITLE
Feature/new image block

### PR DESCRIPTION
**Pull request by gustaveen**
### Updated image block
Now rounded corners, updated max-width and color

Screenshot 2020-03-12 at 17 50 34

### CODE REVIEW

File: storyblok/components.json [hunk 1]
```diff
@@ -1615,6 +1615,49 @@
...
+        },
+        "tab-9266977a-879b-4499-9cef-2e7948d2ca8b": {
+          "type": "tab",
+          "display_name": "Styling",
+          "keys": [
+            "extra_styling"
+          ],
+          "pos": 0
+        },
+        "extra_styling": {
```
Comment (palmenhq) [inline storyblok/components.json]: shall we include this in the block, or remove it?

**Review by palmenhq — COMMENTED** [review]
awesome!

File: storyblok/components.json (hunk 1 above)
Comment (gustaveen) [reply storyblok/components.json]: Let's remove it 👍

**Comment by gustaveen** [comment]
Update now @palmenhq :)

**Review by palmenhq — APPROVED** [review]

**Pull request merged**